controllers/controllers.js

```diff
@@ -68,3 +69,14 @@ exports.getCommentsById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.postComment = (req, res, next) => {
+  const { article_id } = req.params;
+  insertComment(req.body, article_id)
+    .then(([comment]) => {
```

Comment:
Also, if the model provides the comment itself rather than in an array, you wouldn't need to array destructure here